EventFilter/HcalRawToDigi/plugins/PackerHelp.h

```diff
@@ -508,7 +516,7 @@ class UHTRpacker{
     HcalElectronicsId eid(readoutMap->lookup(detid));
     // loop over words in dataframe
     for(edm::DataFrame::iterator dfi=qiedf.begin() ; dfi!=qiedf.end(); ++dfi){      
-      if( dfi >= qiedf.end()-QIE11DataFrame::FLAG_WORDS ){
+      if( dfi >= qiedf.end() ){ //include flag word
```

Comment:
I see changes in the HcalNoise plots. Is this the place in this PR that drives the changes?